File: seed/ts-sdk/examples/retain-original-casing/src/dynamic-snippets/example6/snippet.ts
import { SeedExamplesClient } from "../..";

async function main() {
    const client = new SeedExamplesClient({
        environment: "https://api.fern.com",
        token: "<token>",
    });
    await client.file.service.getFile("filename", {
        "X-File-API-Version": "X-File-API-Version",
    });
}
main();

File: seed/go-sdk/examples/client-name-with-custom-constructor-name/dynamic-snippets/example7/snippet.go
package example

import (
    client "github.com/examples/fern/client"
    option "github.com/examples/fern/option"
    context "context"
)

func do() () {
    client := client.New(
        option.WithBaseURL(
            "https://api.fern.com",
        ),
        option.WithToken(
            "<token>",
        ),
    )
    client.Health.Service.Check(
        context.TODO(),
        "id-2sdx82h",
    )
}
